### PR TITLE
Lightning components

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
Per Kim feedback and "Lightning Components Style Guide" at https://docs.google.com/document/d/1UL9a6cVZLu0lajgWOYiD_tHM8qp4_JRzFjCfGLwv01g/edit , standardize on name and capitalization "Lightning components". Fixed one or two other typos and formatting bugs as well.

### CODE REVIEW

**Review by dukesphere — APPROVED** [review]

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ endif::[]
 
 You design the pages of your API Community Manager (ACM) portal by adding and configuring Lightning components in Community Builder.
 
-These Lightning Components include both the components available in all Salesforce Communities, and the ACM Lightning components only available in ACM.
+These Lightning components include both the Lightning components available in all Salesforce Communities, and the ACM Lightning components only available in ACM.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:9]: Lightning components only available in ACM.
  > Lightning components available only in ACM.

File: modules/ROOT/pages/configure-acm.adoc [hunk 2]
```diff
@@ -172,7 +172,7 @@ NOTE: Remember to associate the external data source with this authentication pr
 . If you don't have an active session in your Anypoint Platform organization, the *Anypoint Login* page is shown. Log in to the Anypoint Platform to complete the authentication. If you already have an active session, authentication is performed without requiring a login.
 . Verify the Exchange *External Data Sources* *Authentication Status* is *Authenticated*.
 
-After authenticating the External Data Source, provide the credentials of an Anypoint Platform system user with read-only access, for components like the API Console which retrieve data directly from the Anypoint Platform APIs.
+After authenticating the External Data Source, provide the credentials of an Anypoint Platform system user with read-only access, for Lightning components like the API Console which retrieve data directly from the Anypoint Platform APIs.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-acm.adoc:175]: This is a complex sentence. Can you reword to simplify or maybe break into two sentences?

Comment (kbostrom) [reply modules/ROOT/pages/configure-acm.adoc:175]: If you use "which," you need to precede it with a comma; if not, use "that".

File: modules/ROOT/pages/configure-acm.adoc [hunk 3]
```diff
@@ -427,11 +427,11 @@ These steps enable guests to register as members in your ACM portal.
 
 ==== Add Self Register Component to Registration Page [[add-self-register-custom-component-to-register-page]]
 
-The self-register component enables guest users to register themselves as members of your community. If you're not using any of the ACM out-of-the-box templates, use these steps to add the ACM-specific self-registration component.
+The self-registration Lightning component enables guest users to register themselves as members of your community. If you're not using any of the ACM out-of-the-box templates, use these steps to add the ACM-specific self-registration Lightning component.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-acm.adoc:430]: any of the provided ACM templates, follow these steps to add...:

File: modules/ROOT/pages/configure-acm.adoc [hunk 4]
```diff
@@ -427,11 +427,11 @@ These steps enable guests to register as members in your ACM portal.
...
 
 . In the ACM control panel, open *Community Builder* and navigate to the *Register* page in the Pages menu on the top left.
-. Remove the standard *Self Registration* component by clicking the delete icon next to it.
-. Navigate to *Components*, search for *Self Register* and add the component to the page.
+. Remove the standard *Self Registration* Lightning component by clicking the delete icon next to it.
+. Navigate to *Components*, search for *Self Register* and add the Lightning component to the page.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-acm.adoc:434]: comma before "and"
GLOBALLY use the Oxford comma

File: modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -33,7 +33,7 @@ Other ACM features enable your organization to engage with partners and ensure t
 
 == Building Your API Community
 
-Personalize your community look and feel, branding, and site structure by using  https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder, role=external, window=blank], a drag-and-drop editor that enables you to create and customize the user experience at every level of detail. Community Builder makes it easy to create a tailored experience using pre-existing components and simple clicks, or using the full flexibility of HTML and CSS for precise pixel-perfect experiences.
+Personalize your community look and feel, branding, and site structure by using  https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder, role=external, window=blank], a drag-and-drop editor that enables you to create and customize the user experience at every level of detail. Community Builder makes it easy to create a tailored experience using pre-existing Lightning components and simple clicks, or using the full flexibility of HTML and CSS for precise pixel-perfect experiences.
```
Comment (kbostrom) [inline modules/ROOT/pages/index.adoc:36]: preexisting

Comment (kbostrom) [reply modules/ROOT/pages/index.adoc:36]: This is pretty full of sales jargon....

File: modules/ROOT/pages/publish-apis.adoc [hunk 6]
```diff
@@ -33,7 +33,7 @@ Now set the API information, including visibility, icon, and description.
 . Select *Replace Icon* to set the API version icon.
 . Enter a *Description* for the API version.
 . Leave the *Visibility* as *Admin Only* while you preview the API version as described in the next section.
-. When your API is displayed on the community site in the API carousel component or in an API card, by default it has a single *API Details* button. It can also have an optional *Learn more* button that links to a business content page. In the *Associated "Learn-more" Page* section, select *None* to keep the default, or select *Custom page* and set the *Custom page URL* to a business content page created in Community Builder.
+. When your API is displayed on the community site in the API Carousel component or in an API card, by default it has a single *API Details* button. It can also have an optional *Learn more* button that links to a business content page. In the *Associated "Learn-more" Page* section, select *None* to keep the default, or select *Custom page* and set the *Custom page URL* to a business content page created in Community Builder.
```
Comment (kbostrom) [inline modules/ROOT/pages/publish-apis.adoc:36]: ditto previous edits to this same text in another file

**Review by kbostrom — COMMENTED** [review]
Here you go!

**Pull request merged**